sktime/forecasting/pytorchforecasting.py

```diff
@@ -115,19 +115,11 @@ class PytorchForecastingTFT(_PytorchForecastingAdapter):
     """  # noqa: E501
 
     _tags = {
-        # packaging info
```

Comment:
why are you removing this, this was informative

Reply:
It's already in adapter class. If we want to modify it, for example changing the version bound of torch, we will have to make modification in multiple places if this is not removed.